server.js

```diff
@@ -49,6 +51,27 @@ server.register({ register: require('./api') }, function(err) {
     });
     throw err;
   }
+
+  server.ext('onPreResponse', function(req, reply) {
```

Comment:
I think I should probably factor this out into another file so I can give it a name.

```js
server.ext('onPreResponse', clearTemporaryFiles);
```

or something...